src/fsharp/fsi/fsi.fs

```diff
@@ -2755,7 +2755,14 @@ type FsiEvaluationSession (fsi: FsiEvaluationSessionHostConfig, argv:string[], i
     let currentDirectory = Directory.GetCurrentDirectory()
     let tryGetMetadataSnapshot = (fun _ -> None)
 
-    let defaultFSharpBinariesDir = FSharpEnvironment.BinFolderOfDefaultFSharpCompiler(FSharpEnvironment.tryCurrentDomain()).Value
+    let defaultFSharpBinariesDir = 
+        try
+            FSharpEnvironment.BinFolderOfDefaultFSharpCompiler(FSharpEnvironment.tryCurrentDomain()).Value
+        with
+        | _ ->
+            let d = System.Reflection.Assembly.GetExecutingAssembly() //IO.Directory.GetCurrentDirectory()        
+            let fi = IO.FileInfo(d.Location)
+            fi.DirectoryName
```

Comment:
You could directly use `Path.GetDirectoryName` here; there is no reason to create a `FileInfo`.

Reply:
Thanks! I changed it!!

Reply:
Oops, the new commit is created, however the PR hasn't yet updated.... should I close it and recreate a new PR?